### ISSUE TITLE
Duplicate class

### ISSUE THREAD

**Issue by ubuntudroid** [adjust/android_sdk#561]
The 4.33.1 SDK version seems to introduce a new class which conflicts with other classes due to its super generic package and class name:

```
Duplicate class a.a found in modules adjust-android-4.33.1-runtime (com.adjust.sdk:adjust-android:4.33.1) and core-3.4-release.2-release-runtime (io.purchasely:core:3.4.2)
```

Doesn't happen with 4.33.0.

The Purchasely SDK obviously does something equally problematic (and I've reached out to them with the same request), but I'm pretty sure it may also easily happen with many other SDKs.

It would be great if you could at least change the package of this class to be something less generic, so that the likelyhood of duplicate class errors is minimized.

**Comment by shashanksu** [comment]
Hi @ubuntudroid,
Thank you for reporting this.  Indeed, we could see this `a.a` package bundled in version 4.33.1.
Will fix it in next version and update here. 
Thanks again one more time for letting us know.
Cheers

**Comment by shashanksu** [comment]
Hi @ubuntudroid,
We have released a new version 4.33.2 with the fix.  
Please let us know in case you still see the issue.
Thank you.

**Comment by ubuntudroid** [comment]
@shashanksu with 4.33.2 the issue is gone, many thanks! 🙏
